core-plugins/docs/XMLReader-batchsource.md

```diff
@@ -0,0 +1,107 @@
+# XMLReader Batch Source
+
+
+Description
+-----------
+The XML Reader plugin is a source plugin that allows users to read XML files stored on HDFS.
+
+
+Use Case
+--------
+Customer XML feeds that are dropped onto HDFS. These can be small to very large XML  files. The files have to read,
+parsed and when used in conjunction with XML Parser they are able to extract fields. This reader will emit one XML
+event that is specified by a Node Path to be extracted from the file.
+
+
+Properties
+----------
+**referenceName:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**path:** Path to file(s) to be read. If a directory is specified, terminate the path name with a '/'.
+
+**nodePath:** Node path to emit individual event from the schema.
+Example - '/book/price' to read only price under the book node
+
+**pattern:** Pattern to select specific file(s).
+Example -
+1. Use '^' to select file with name start with 'catalog', like '^catalog'.
+2. Use '$' to select file with name end with 'catalog.xml', like 'catalog.xml$'.
+3. Use '*' to select file with name contains 'catalogBook', like 'catalogBook*'.
```

Comment:
The asterisk needs to be escaped in Markdown:
```
3. Use '\*' to select file with name contains 'catalogBook', such 'catalogBook*'.
```
We want:

3. Use '\*' to select file with name contains 'catalogBook', such as 'catalogBook*'.

not

3. Use '*' to select file with name contains 'catalogBook', like 'catalogBook*'.